tests/internal/scripts/ccuda/cuda_array_managed.py

```diff
@@ -11,7 +11,7 @@ def square(a):
     threads_per_block = 5
     n_blocks = 1
     a = cuda.array([0,1,2,3,4], memory_location = 'managed')
-    cuda.synchronize()
-    square[n_blocks, threads_per_block](a)
-    cuda.synchronize()
+    cuda.deviceSynchronize()
+    square[5, 1](a)
+    cuda.deviceSynchronize()
```

Comment:
the naming changed in #1274 from `deviceSynchronize` to `synchronize`

Reply:
Fixed in [8ef5990](https://github.com/pyccel/pyccel/pull/1235/commits/8ef59906f98230ce1b03545e6817e8279ce751d3).